src/promptflow/tests/conftest.py

```diff
@@ -8,14 +8,14 @@
 
 import pytest
 from _constants import (
+    CLI_PERF_MONITOR_AGENT,
```

Comment:
I understand this change is done by pre-commit? if so, maybe we still miss some steps in dev setup...

Reply:
Yep